docs/documentation/case.md

```diff
@@ -677,7 +677,7 @@ The parameters are optionally used to define initial velocity profiles and pertu
 
 - `vel_profile` activates setting the mean streamwise velocity to hyperbolic tangent profile. This option works only for 2D and 3D cases.
```

Comment:
this line of documentation needs more details. Hyperbolic tangent with what parameters? it also doesn't mention anything about mixing layers.